querydsl-sql-spatial/src/main/java/com/querydsl/sql/spatial/SQLServerGeometryType.java

```diff
@@ -72,7 +74,7 @@ public String getLiteral(Geometry geometry) {
         if (geometry.getSRID() > -1) {
             return "geometry::STGeomFromText('" + str + "', " + geometry.getSRID() + ")";
         } else {
-            return "geometry::STGeomFromText('" + str + "')";
+            return "geometry::STGeomFromText('" + str + "', 4326)";
```

Comment:
The SRID 4326 stands for WGS 84 http://en.wikipedia.org/wiki/World_Geodetic_System#A_new_World_Geodetic_System:_WGS_84

It is used as the default SRID elsewhere https://msdn.microsoft.com/en-us/library/bb933824.aspx